subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -173,24 +173,29 @@ if NRF_CLOUD_AGPS
 config NRF_CLOUD_AGPS_AUTO
 	bool "Automatically request A-GPS on bootup"
 
-config NRF_CLOUD_AGPS_SINGLE_CELL_ONLY
+endif # NRF_CLOUD_AGPS
+
+config NRF_CLOUD_CELL_POS
+	bool "Enable nRF Cloud Cellular Positioning"
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_NETWORK
+
+if NRF_CLOUD_CELL_POS
+
+config NRF_CLOUD_CELL_POS_SINGLE_CELL_ONLY
```

Comment:
This is a bit odd option. It's off by default, but it's the only cell positioning method currently supported. What happens if NRF_CLOUD_CELL_POS is enabled without NRF_CLOUD_CELL_POS_SINGLE_CELL_ONLY? Just wondering if we should remove it until we have support for both single and multi cell positioning.

Reply:
Will change following nrf_cloud update

Reply:
Yes, since cell pos is being pulled out of GPS/AGPS, i don't think `NRF_CLOUD_CELL_POS_SINGLE_CELL_ONLY` is necessary.  
Cell pos probably doesn't belong inside this A-GPS menu either?

Reply:
will update nrf_cloud lib according to your description